source3/libsmb/libsmb_setget.c

```diff
@@ -98,6 +98,19 @@ smbc_setDebug(SMBCCTX *c, int debug)
 	TALLOC_FREE(frame);
 }
 
+/** set configuration file */
+void smbc_setConfiguration(SMBCCTX *c, const char* file)
```

Comment:
Please use an error return here. E.g. ret = -1, errno = ENOENT if the file doesn't exist.
Also, please use consistent style - const char *file